mocha/mocha-node.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for mocha 2.2.5
```

Comment:
Could you add a note here to make it obvious why there are two separate versions of the typings? Perhaps something like:
```TypeScript
// Type definitions for mocha 2.2.5 (running on node.js)
```
And in `mocha.d.ts`
```TypeScript
// Type definitions for mocha 2.2.5 (running in a browser)
```